apps/docs/content/components/Grid.mdx

```diff
@@ -0,0 +1,274 @@
+
+The `Grid` component uses a 12-column layout system. Use `span` to control the number of columns that should be displayed.
+
+```jsx live
+<Grid enableOverlay>
+  <Grid.Column span={1}></Grid.Column>
+  <Grid.Column span={11}></Grid.Column>
+  <Grid.Column span={2}></Grid.Column>
+  <Grid.Column span={10}></Grid.Column>
+  <Grid.Column span={3}></Grid.Column>
+  <Grid.Column span={9}></Grid.Column>
+  <Grid.Column span={4}></Grid.Column>
+  <Grid.Column span={8}></Grid.Column>
+  <Grid.Column span={5}></Grid.Column>
+  <Grid.Column span={7}></Grid.Column>
+  <Grid.Column span={6}></Grid.Column>
+  <Grid.Column span={6}></Grid.Column>
+  <Grid.Column span={7}></Grid.Column>
+  <Grid.Column span={5}></Grid.Column>
+  <Grid.Column span={8}></Grid.Column>
+  <Grid.Column span={4}></Grid.Column>
+  <Grid.Column span={9}></Grid.Column>
+  <Grid.Column span={3}></Grid.Column>
+  <Grid.Column span={10}></Grid.Column>
+  <Grid.Column span={2}></Grid.Column>
+</Grid>
+```
+
+### Nested grids
+
+```jsx live
+<Grid enableOverlay>
+  <Grid.Column>
+    <Grid enableOverlay>
+      <Grid.Column span={4}></Grid.Column>
+      <Grid.Column span={4}></Grid.Column>
+      <Grid.Column span={4}></Grid.Column>
+    </Grid>
+  </Grid.Column>
+</Grid>
+```
+
+### Responsive behavior
+
+Use `start` with an `Object` of breakpoint-specific keys and `number` values to enable responsive behavior.
+
+```jsx live
+<Grid enableOverlay>
+  <Grid.Column
+    span={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+</Grid>
+```
+
+### Column offset
+
+Use `start` to provide a positioning offset.
+
+```jsx live
+<Grid enableOverlay>
+  <Grid.Column span={4}></Grid.Column>
+  <Grid.Column start={8}></Grid.Column>
```

Comment:
Nothing against the prop name `start`, but is there a reason for not keeping it consistent with the widely adopted `offset`?